app/grandchallenge/components/models.py

```diff
@@ -1143,6 +1151,15 @@ def _clean_store_in_database(self):
                 f"Interface {self.kind} objects cannot be stored in the database"
             )
 
+    def _validate_example_value(self):
+        if self.example_value:
+            if self.schema:
+                JSONValidator(schema=self.schema)(value=self.example_value)
+            else:
+                raise ValidationError(
+                    "Schema is not provided to validate the example_value against"
```

Comment:
Setting a schema is not necessary for value validation.

Reply:
Please correct me if I am worng. We want to valiadte the example_value against the custom schema before saving the model and not just checking that the example_value is a valid JSON, right? This is what this call is doing right now, and in the GC Admin you see a response like this:

![Screenshot 2024-08-14 at 14 29 49](https://github.com/user-attachments/assets/e18f1b90-4a5b-4c94-bb85-e218e7dd979d)

Reply:
Yes. But setting the schema is not required.